data-prepper-plugins/geoip-processor/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1,3 @@
+# To enable mocking of final classes with vanilla Mockito
```

Comment:
Why do we need this?

Reply:
"mock-maker-inline" is required to mock the final classes. For example AWS "Region" class is final class.